.github/tailor.yaml

```diff
@@ -2,9 +2,9 @@ rules:
   - name: commit title
     description: all commit titles must have a scope, be lowercase, and be no more than 50 characters
     expression:  |-
-      .commits all(((.message test "^[a-z/*-]+: [[:alnum:] -'./-]+") and (.message test "^.{0,50}\\n")) or (.message test "^Revert"))
+      .commits all(((.title test "^[a-z/*-]+: [[:alnum:] -'./-]+$") and (.title length < 51)) or (.title test "^Revert"))
```

Comment:
Is `.title` new to tailor? It's not under the [root context](https://github.com/crawford/tailor#root-context).

Reply:
https://github.com/coreos/tailor#root-context